src/foam/nanos/crunch/services.jrl

```diff
@@ -61,11 +61,14 @@ p({
   "serve": true,
   "description": "DAO responsible for storing capabilities' categories.",
   "serviceScript": """
+    value = {foam.nanos.crunch.CapabilityCategoryCapabilityJunction.SOURCE_ID,foam.nanos.crunch.CapabilityCategoryCapabilityJunction.TARGET_ID};
+    
     return new foam.dao.EasyDAO.Builder(x)
       .setAuthorizer(new foam.nanos.auth.GlobalReadAuthorizer("capabilityCategoryCapabilityJunction"))
       .setJournalType(foam.dao.JournalType.SINGLE_JOURNAL)
       .setJournalName("capabilityCategoryCapabilityJunction")
       .setOf(foam.nanos.crunch.CapabilityCategoryCapabilityJunction.getOwnClassInfo())
+      .setIndex(value)
```

Comment:
Isn't that the primary key, so it would be indexed already anyway?

Ex.:
> foam.nanos.crunch.CapabilityCategoryCapabilityJunction.ID.propNames;
(2) ["sourceId", "targetId"]

Test if just adding an index for targetId helps, otherwise, I don't think this helps.